pkg/website/generated.go.txt

```diff
@@ -75,6 +75,7 @@ exampleSets = [
           {"id": "example-edit-map-value"},
           {"id": "example-remove-map-item"},
           {"id": "example-rename-key-in-map"},
+          {"id": "example-append-map-to-array"},
```

Comment:
This example may follow logically after `Append array item`, how would you feel about this example being number 11?